### PR TITLE
PWGHF: Adapt selectors/tasks with 2D configurables

### PR DESCRIPTION
**Pull request by aalkin**

### CODE REVIEW

**Comment by aalkin** [comment]
@vkucera @ginnocen here's the PR (for some reason I cannot add reviewers directly, so I'm pinging you in a comment).

File: Analysis/Core/include/AnalysisCore/HFSelectorCuts.h [hunk 1]
```diff
@@ -19,7 +19,9 @@ namespace pdg
 {
 enum code {
   kD0 = 421,
-  kD0bar = -421
+  kD0bar = -421,
+  kDPlus = 411,
+  kLambdaC = 4122
```
Comment (vkucera) [inline Analysis/Core/include/AnalysisCore/HFSelectorCuts.h]: Please add `kXiC`.

**Review by vkucera — DISMISSED** [review]

**Comment by vkucera** [comment]
@aalkin Can you fix the mismatch between `ptBins` in selectors and `pTBins` in tasks?

**Comment by aalkin** [comment]
> Can you fix the mismatch between `ptBins` in selectors and `pTBins` in tasks?

Done. Now `pTBins` is used throughout.

**Review by vkucera — APPROVED** [review]
@ginnocen

**Review by ginnocen — APPROVED** [review]

**Comment by ginnocen** [comment]
@jgrosseo @ktf @iarsene. Could you please merge this one?

**Review by iarsene — APPROVED** [review]

**Pull request merged**